packages/pharos/src/components/checkbox/PharosCheckbox.react.stories.mdx

```diff
@@ -140,25 +140,25 @@ export const Template = (args) => (
   </Story>
 </Canvas>
 
-# Dark
+# On background
 
 <Canvas withToolbar>
-  <Story name="Dark">
+  <Story name="On background">
     {() => {
       return (
         <div>
-          <div className="checkbox-example__container--dark">
-            <PharosCheckbox name="dark" dark>
+          <div className="checkbox-example__container--on-background">
```

Comment:
Think you'll need to rename `checkbox-example__container--dark` defined in `pharos-site/static/styles/global.scss`

Reply:
Hmm, must've been in the wrong directory when I searched for that.